how-to/deploy/orchestrated/ansible.md

```diff
@@ -616,8 +616,9 @@ To enable the following control variables, use the capitalized `True`. To disabl
 | Variable Name | Description |
 | :---- | :------- |
 | cluster_name | the name of a cluster, adjustable |
```

Comment:
How about enclosing the variables with ``` ` ```?
```suggestion
| `cluster_name` | the name of a cluster, adjustable |
```

Reply:
Strictly, parameter names should be enclosed with ``. But none of the parameters in this document table are so enclosed.

Reply:
OK